WhisperAPI/WhisperAPI/Controllers/VersionController.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace WhisperAPI.Controllers
+{
+    [Route("/Whisper/[Controller]")]
+    public class VersionController : Controller
+    {
+        [HttpGet]
+        public IActionResult GetVersion()
+        {
+            return this.Ok("{\"version\": \"11\"}");
```

Comment:
Make a new object model Version containing a string and return it instead of this hardcoded string jsonify